charmcraft/parts.py

```diff
@@ -184,6 +184,15 @@ def get_build_packages(self) -> Set[str]:
 
     def get_build_environment(self) -> Dict[str, str]:
         """Return a dictionary with the environment to use in the build step."""
+        try:
+            os_release = os_utils.OsRelease()
```

Comment:
Please move this outside the `try` since it doesn't throw any of the expected errors

Reply:
Actually, I'm retracting that.